repairnator/src/main/java/fr/inria/spirals/repairnator/process/inspectors/ProjectInspector.java

```diff
@@ -51,16 +52,15 @@
     private File[] repairSourceDir;
     private boolean isReproducedAsFail;
     private boolean isReproducedAsError;
-    protected Build previousBuild;
     protected boolean previousBuildFlag;
 
-    public ProjectInspector(Build failingBuild, String workspace, List<AbstractDataSerializer> serializers,
-            String nopolSolverPath, boolean push, RepairMode mode) {
-        this.build = failingBuild;
+    public ProjectInspector(BuildToBeInspected buildToBeInspected, String workspace, List<AbstractDataSerializer> serializers,
+                            String nopolSolverPath, boolean push, RepairMode mode) {
+        this.buildToBeInspected = buildToBeInspected;
```

Comment:
Ok. For RepairNator inspector, I tested if the status of the buildToBeInspected is `ScannedBuildStatus.ONLY_FAIL`, which is the interesting status for RepairNator, but there is a moment I assigned the status `ScannedBuildStatus.UNKNOWN` when RepairNator is running (method `readBuildFromInput`, class `ProjectScanner`). I don't know if the builds with `ScannedBuildStatus.UNKNOWN` status is also interesting for you to be inspect. Is it?